app/helpers/proxmox_vm_cloudinit_helper.rb

```diff
@@ -114,4 +119,25 @@ def attach_cloudinit_iso(node, iso)
   def default_iso_path
     "/var/lib/vz/template/iso"
   end
+
+  def update_boot_order(image_id)
+    vm = find_vm_by_uuid(image_id)
+    return if vm.disks.nil?
+    disks = vm.disks.map { |disk| disk.split(":")[0] }.join(";")
+    { boot: "order=" + disks }
```

Comment:
how is the boot order, ordered?

Reply:
Since vm boots from one of the disks of image/template. We don't know which one of the disks is bootdisk so add all of them to boot order and vm will boot from one of them which is bootdisk. If we don't do this then it will try to boot from its own  network or iso image (cd-rom) which is just a cloudinit disk.